src/pages/GlobalPage.js

```diff
@@ -82,9 +85,15 @@ function GlobalPage() {
                         <div />
                       </RowBetween>
                       <RowBetween align="flex-end">
-                        <TYPE.main fontSize={'1.5rem'} lineHeight={1} fontWeight={600}>
-                          {formattedNum(oneDayVolumeUSD, true)}
-                        </TYPE.main>
+                        {oneDayVolumeUSD ? (
+                          <TYPE.main fontSize={'1.5rem'} lineHeight={1} fontWeight={600}>
+                            {formattedNum(oneDayVolumeUSD, true)}
+                          </TYPE.main>
+                        ) : (
+                          <div style={{ width: '35px', margin: '0px 0px 0px 16px' }}>
+                            <div className="dot-flashing"></div>
+                          </div>
+                        )}
```

Comment:
Also here, in mobile view, we can hide the percentage change while loading
![image](https://user-images.githubusercontent.com/9011637/163794775-4e6b5b47-d4fd-4955-b92c-e93181d4e0ff.png)